### PR TITLE
Dialog width fixes for umbraco 7.5.4

### PR DESCRIPTION
**Pull request by tompipe**
Quick fixes for dialog width in umbraco 7.5.4.

#worksforme #notcrossbrowsertested

### CODE REVIEW

File: src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.dialog.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<ng-form name="dtgeForm" novalidate>
+﻿<ng-form id="dtgeForm" name="dtgeForm" novalidate>
```
Comment (leekelleher) [inline src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.dialog.html]: Is `id="dtgeForm"` needed? Can't see it being used?

Comment (tompipe) [inline src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.dialog.html]: ah just spotted that myself, i added it earlier whilst trying to make it work, and found a better way. You're right, its not needed now

**Comment by tompipe** [comment]
Just a thought, this wider dialog may look a bit odd if the doctype hasn't got a RTE on it, and only has narrow properties like textstrings/checkboxes etc.

Could we evaluate the contents, determine the max width and resize the dialog dynamically?

**Comment by leekelleher** [comment]
@tompipe Crazy that it's been a year since you made this PR! 🙄
I'm finally giving DTGE a bit of love and reviewing this.

I'm trying it out against Umbraco 7.6.11 and it appears that if I set the `modalClass` as `"umb-modal wide"`, then it will open at 640px wide.  I'm not sure how long the "`wide`" class has been in Umbraco, but I think it would cover this request, (without the extra CSS).

Personally I think the wider dialog/modal looks fine with non-RTE, just textbox, checkbox, dropdown fields.  But I'm not sure whether this should be an optional config setting? // @mattbrailsford - would be good to have your input on this one.

We could either hard-code, `modalClass: "umb-modal wide dtge-modal"` in doctypegrideditor.services.js, or have an option in the config to enable it to be wider (or maybe override the modalClass value?

**Comment by mattbrailsford** [comment]
I vote "wide" config option

**Comment by alindgren** [comment]
Searching Umbraco repo, unless I'm reading it wrong, it looks like the `wide` class has been there a long time:

https://github.com/umbraco/Umbraco-CMS/blame/e0025db56d52b770d2b3aedbd48a3b804fd15ef0/src/Umbraco.Web.UI.Client/src/less/modals.less#L179-L182

I'd love to see wide as an option, or maybe even could be the default.

**Comment by leekelleher** [comment]
Closing this PR off, as PR #86 has replaced the dialog modal with the umb-overlay directive.
I'd still like to have a "wide" config option - if anyone is up for it?

@tompipe - apologies again for delays with feedback this PR.

**Pull request closed without merging**